CHANGELOG.md

```diff
@@ -1 +1,164 @@
+- buildkit: inline cache: fix handling of duplicate blobs [moby/moby#41234](https://github.com/moby/moby/pull/41234) [moby/buildkit#1568](https://github.com/moby/buildkit/pull/1568)
+  - Fixes https://github.com/moby/buildkit/issues/1388 cache-from working unreliably
+  - Fixes https://github.com/moby/moby/issues/41219 Image built from cached layers is missing data
+* Allow ssh:// for remote context URLs [moby/moby#40179](https://github.com/moby/moby/pull/40179)
+* builder: remove legacy build's session handling (was experimental) [moby/moby#39983](https://github.com/moby/moby/pull/39983)
+
+### Client
+
++ Add swarm jobs support to CLI [docker/cli#2262](https://github.com/docker/cli/pull/2262)
++ Add capabilities support to stack/service commands [docker/cli#2687](https://github.com/docker/cli/pull/2687)
++ Add `-a/--all-tags` to docker push [docker/cli#2220](https://github.com/docker/cli/pull/2220)
++ Add support for Kubernetes username/password auth [docker/cli#2308](https://github.com/docker/cli/pull/2308)
++ Add `--pull=missing|always|never` to `run` and `create` commands [docker/cli#1498](https://github.com/docker/cli/pull/1498)
++ Add `--env-file` flag to `docker exec` for parsing environment variables from a file [docker/cli#2602](https://github.com/docker/cli/pull/2602)
++ Add shorthand `-n` for --tail option [docker/cli#2646](https://github.com/docker/cli/pull/2646)
++ Add log-driver and options to service inspect "pretty" format [docker/cli#1950](https://github.com/docker/cli/pull/1950)
++ docker run: specify cgroup namespace mode with `--cgroupns` [docker/cli#2024](https://github.com/docker/cli/pull/2024)
++ `docker manifest rm` command to remove manifest list draft from local storage [docker/cli#2449](https://github.com/docker/cli/pull/2449)
+* Add "context" to "docker version" and "docker info" [docker/cli#2500](https://github.com/docker/cli/pull/2500)
+* Propagate platform flag to container create API [docker/cli#2551](https://github.com/docker/cli/pull/2551)
+* The `docker ps --format` flag now has a `.State` placeholder to print the container's state without additional details about uptime and health check [docker/cli#2000](https://github.com/docker/cli/pull/2000)
+* Add support for docker-compose schema v3.9 [docker/cli#2073](https://github.com/docker/cli/pull/2073)
+* Add support for docker push --quiet [docker/cli#2197](https://github.com/docker/cli/pull/2197)
+* Hide flags that are not supported by BuildKit, if BuildKit is enabled [docker/cli#2123](https://github.com/docker/cli/pull/2123)
+* Update flag description for `docker rm -v` to clarify the option only removes anonymous (unnamed) volumes [docker/cli#2289](https://github.com/docker/cli/pull/2289)
+* Improve tasks printing for docker services [docker/cli#2341](https://github.com/docker/cli/pull/2341)
+* Use ssh multiplexing [docker/cli#2132](https://github.com/docker/cli/pull/2132)
```

Comment:
I think this one was reverted: https://github.com/docker/cli/pull/2303 :sweat_smile: